terraform/modules/kubernetes/variables.tf

```diff
@@ -0,0 +1,41 @@
+variable "app_environment" {}
+variable "namespace" {}
+variable "app_docker_image" {}
+variable "app_environment_variables" {}
+variable "app_secrets" {}
+variable "cluster" {}
+variable "webapp_startup_command" {}
+
+locals {
+  app_config_name        = "apply-config-${var.app_environment}"
+  app_secrets_name       = "apply-secrets-${var.app_environment}"
+  database_url           = "postgres://postgres:password@${local.postgres_service_name}:5432/${local.postgres_service_name}"
```

Comment:
This depends on `local.postgres_service_name}`, it would make sense to move it after it